tests/components/advantage_air/test_sensor.py

```diff
@@ -57,7 +66,6 @@ async def test_sensor_platform(hass, aioclient_mock):
         {ATTR_ENTITY_ID: [entity_id], ADVANTAGE_AIR_SET_COUNTDOWN_VALUE: value},
         blocking=True,
     )
-    assert len(aioclient_mock.mock_calls) == 3
```

Comment:
Why were this assert and the one below removed?

Reply:
Re-enabling this entity made the count inconsistent. Checking the number of API calls didn't really prove or test anything either.